StRoot/StMcEventMaker/StMcEventMaker.cxx

```diff
@@ -1067,12 +1069,14 @@ Int_t StMcEventMaker::Make()
 		long nPseudoPadrow = 0;
 		long ihit;
 		for(ihit=0; ihit<NHits; ihit++) {
-		    if (tpcHitTable[ihit].volume_id < 101 || tpcHitTable[ihit].volume_id > 2445) {
-			if (tpcHitTable[ihit].volume_id <= 202445 &&
-			    tpcHitTable[ihit].volume_id > 2445) nPseudoPadrow++; 
+#if 0 /* keep all tpc hits => Inner TPC sector upgrade */
```

Comment:
The line 1072 should be changed to 
#if 1

Reply:
Changing it to `#if 1` is equivalent to removing `#if 0` and `#endif` completely

Reply:
Hi Yuri/Xianglei, shall we resolve this line and try to merge this request in asap? We also need the commit to patch the SL21c library as well (I see Xianglei submitted a new request, need to be sure they are consistent).

THanks

Reply:
In PR289, this line has been changed to 
#if 1
The reason is shown in slide# 3 of
https://drupal.star.bnl.gov/STAR/system/files/embedding_20220119.pdf

Reply:
I removed this line from Yuri's branch. If no further comment, I would suggest we merge this into main.